### PR TITLE
docs: add jobs

### PR DESCRIPTION
**Pull request by ritazh**
Signed-off-by: Rita Zhang <rita.z.zhang@gmail.com>

### CODE REVIEW

File: jobs.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+### Engineering
+#### Secure supply chain
+- [Software Engineer II, Atlanta, remote](https://careers.microsoft.com/us/en/job/1210002/Software-Engineer-II)
+- [Senior Software Engineer, Atlanta, remote](https://careers.microsoft.com/us/en/job/1309571/Senior-Software-Engineer)
+
+#### Cloud native upstream
+- [Software Engineer II, Atlanta, remote](https://careers.microsoft.com/us/en/job/1224166/Software-Engineer-II)
+
+#### WASM
```
Comment (devigned) [inline jobs.md:17]: These two can be dropped. I'll add when new openings are available.

**Review by devigned — CHANGES_REQUESTED** [review]

File: jobs.md (hunk 1 above)
Comment (ritazh) [reply jobs.md:17]: done

File: jobs.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+Come and work with us!
+
+## Upstream team
+
+Interested in working on open source and containers at a leading hyper-scale cloud? You can help build the future of Kubernetes at Microsoft!  
+
+### Engineering
+#### Secure supply chain
+- [Software Engineer II, Atlanta, remote](https://careers.microsoft.com/us/en/job/1210002/Software-Engineer-II)
+- [Senior Software Engineer, Atlanta, remote](https://careers.microsoft.com/us/en/job/1309571/Senior-Software-Engineer)
```
Comment (sozercan) [inline jobs.md:12]: `Atlanta or US remote`?

Comment (ritazh) [reply jobs.md:12]: This means it's opened for Atlanta but can be US remote. Does that help?

**Review by sozercan — APPROVED** [review]

**Pull request merged**